src/tests/Microsoft.PowerFx.Core.Tests/TestRunnerTests/TestRunnerTests.cs

```diff
@@ -337,13 +322,9 @@ public void TestRunnerError()
         [Fact]
         public void TestRunnerErrorKindMatching()
         {
-            var errorValue = new ErrorValue(
-                IR.IRContext.NotInSource(FormulaType.Number),
-                new ExpressionError { Kind = ErrorKind.InvalidFunctionUsage });
-            var runner = new MockRunner
-            {
-                _hook = (expr, setup) => errorValue // error
-            };
+            var errorValue = new ErrorValue(IR.IRContext.NotInSource(FormulaType.Number), new ExpressionError { Kind = ErrorKind.InvalidFunctionUsage });
+            
+            var runner = new MockRunner { _hook = (expr, setup) => errorValue /* error */ };
```

Comment:
>/* [](http://example.com/codeflow?start=78&length=2)

nit - we should avoid /* comments. 
(I thought we actually had a style cop rule for this)